lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -578,6 +582,84 @@ def get(self, request):
         }
         return render_to_response(self.template_name, context)
 
+    def get_current_tabs(self, course_id):
```

Comment:
This file is also starting to grow pretty large (>1000 lines) and will presumably continue to do so, given our recent emphasis on these kinds of commands. I'd like it if we could come up w/ a strategy to start splitting out this logic.

We could start with a `sysadmin_course_tabs.py` and/or a `sysadmin/` directory to house all of these "extensions". I could image the new file exposing a single, public function, something like:

```python
def process_request(action, request):
    return message
```

Then we'd invoke it with something like:
```python

if action in ['get_current_tabs', 'delete_tab', 'insert_tab']:
    message = sysadmin_course_tabs.process_request(action, request)
```

Let me know what you think.